system/api-server/src/file-upload/file-upload.controller.ts

```diff
@@ -38,7 +38,7 @@ export class ImageUploadController {
   @ApiBody({
```

Comment:
It'd be great to see some comments in these controllers to know what the app is doing

Reply:
Ok I will add comments to the controller and service